models/models.js

```diff
@@ -7,14 +7,18 @@ const fetchTopics = () => {
   });
 };
 
-// Join tables, aggregate function: count()
-// Count comments attached to each 
-// As = comment_count
 const fetchArticles = () => {
   const queryString = `SELECT articles.*, COUNT(comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY created_at DESC`;
   return db.query(queryString).then((result) => {
     return result.rows;
   });
 };
 
-module.exports = { fetchTopics, fetchArticles };
+const fetchSingleArticle = (id) => {
+  const queryString = `SELECT * FROM articles WHERE article_id = ${id}`
+  return db.query(queryString).then((result) => {
+    return result.rows[0];
+  })
```

Comment:
This sql statement is open to sql injection, this should be done using a parameterized query.